tox.ini

```diff
@@ -123,9 +125,6 @@ commands =
         --pull --rm \
         {posargs:-t ansible/molecule:$TAG} .\
     '
-whitelist_externals =
-    sh
```

Comment:
Why did you do this?

Reply:
Only this env needs `sh` so there's no need to move it to the global one.

Reply:
To simplify setup. What is the great feature of keeping these in several places? ... unless we do take a special pleasure in having more complex tox.ini files. Clearly is not related to security or so, in the end any code change would need to be reviewed.

Reply:
This is related to envs being obvious and avoiding a godobject antipattern.

When you look at the env you shouldn't have to look up all the other envs to guess what else may influence it. Explicit is better than implicit.

Also, dumping everything into one env makes it unreadable/unmaintainable as I've explained somewhere else already.